src/generators/nodes/Element.ts

```diff
@@ -216,7 +216,9 @@ export default class Element extends Node {
 		if (this._needsCssAttribute && !this.generator.customElement) {
 			if (!this.attributes.find(a => a.type === 'Attribute' && a.name === 'class')) {
 				block.builders.hydrate.addLine(
-					`${name}.className = "${this.generator.stylesheet.id}";`
+					this.namespace === namespaces.svg
```

Comment:
Elsewhere I think we always use `setAttribute` if any namespace is set at all, not just if the SVG one is set. I'm really not sure what the practical difference is, but do we want to do the same thing here?

Reply:
ah, you're right — I don't think it does make any practical difference (idk, MathML?) but consistency is valuable. updated